patches/mario-kart-8/create_server_config.patch

```
@@ -0,0 +1,104 @@
++
++        # These gRPC credentials are for the server we're implementing
++        self.mario_kart_8_grpc_host = "localhost"
++        self.mario_kart_8_grpc_port = int(os.getenv('PN_MK8_GRPC_PORT', 50051))
++        self.mario_kart_8_grpc_api_key = os.getenv('PN_MK8_GRPC_API_KEY', 'abcdefghijklmnopqrstuvwxyz123456789')
++
++        self.account_database = os.getenv('PN_MK8_ACCOUNT_DATABASE', 'pretendo_account')
++
++        self.pnid_collection = "pnids"
++        self.nex_account_collection = "nexaccounts"
++
++        self.game_db_server = MongoCredentials(
++            host=os.getenv('PN_MK8_MONGO_HOST', '222.222.222.222'),
++            port=int(os.getenv('PN_MK8_MONGO_PORT', 1004))
++        )
++
++        self.game_database = "mk8"
++
++        self.sequence_collection = "counters"
++        self.gatherings_collection = "gatherings"
++        self.sessions_collection = "sessions"
++        self.tournaments_collection = "tournaments"
++        self.tournaments_score_collection = "tournaments_scores"
++        self.ranking_common_data_collection = "commondata"
++        self.rankings_score_collection = "rankings"
++        self.secure_reports_collection = "secure_reports"
++        self.datastore_collection = "datastore"
++        self.restriction_collection = "restrictions"
++
++        self.s3_endpoint_domain = os.getenv('PN_MK8_CONFIG_S3_ENDPOINT', 'minio.pretendo.cc')
++        self.s3_endpoint = "http://" + self.s3_endpoint_domain
++        self.s3_access_key = os.getenv('PN_MK8_CONFIG_S3_ACCESS_KEY', 'minio_pretendo')
++        self.s3_secret = os.getenv('PN_MK8_CONFIG_S3_SECRET', '...')
++        self.s3_region = "it"
++        self.bucket_name = os.getenv('PN_MK8_CONFIG_S3_BUCKET', 'amkj')
++
++        self.redis_uri = "redis://" + os.getenv('PN_MK8_REDIS_URI', '53.53.53.53:1005')  # redis://HOST[:PORT][?db=DATABASE[&password=PASSWORD]]
```

This looks great! Nice way to grab the server configuration from Docker environment variables even though the original server doesn't support it.

I'm not sure that it makes sense to provide default values to `getenv`, though, when those defaults are basically nonsense (like the invalid IPs and secrets). It might be better to just let them be None and have the server fail fast in case of accidentally-missing variables.